File: app/src/main/java/com/hazizz/droid/Communication/POJO/Response/POJOerror.java
package com.hazizz.droid.Communication.POJO.Response;

import lombok.Data;

@Data
public class POJOerror{

    private String time;
    private int errorCode;
    private String title;
    private String message;

    public POJOerror(String time, int errorCode, String title, String message){
        if(time == null){time = "null";}else{this.time = time;}
        this.errorCode = errorCode;
        if(title == null){title = "null";}else{this.title = title;}
        if(message == null){message = "null";}else{this.message = message;}

    }
}

File: lib/communication/pojos/PojoError.dart
import 'package:json_annotation/json_annotation.dart';

import 'Pojo.dart';

part 'PojoError.g.dart';

@JsonSerializable()
class PojoError extends Pojo{
  PojoError({
    this.time,
    this.errorCode,
    this.title,
    this.message,
    this.url,
  });

  final String time;
  final int errorCode;
  final String title;
  final String message;
  final String url;

  factory PojoError.fromJson(Map<String, dynamic> json) =>
      _$PojoErrorFromJson(json);

  Map<String, dynamic> toJson() => _$PojoErrorToJson(this);

}